### PR TITLE
jar, helm, kill: add pt_BR translations

### PR DESCRIPTION
**Pull request by caduvieira**
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages.pt_BR/common/helm.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+`helm repo list`
+
+- Atualiza os repositórios helm:
+
+`helm repo update`
+
+- Remova um repositório helm:
+
+`helm repo remove {{repo_name}}`
```
Comment (navarroaxel) [inline pages.pt_BR/common/helm.md:24]: Please translate the arguments for get this easier for non-English speakers in all page.

**Review by navarroaxel — CHANGES_REQUESTED** [review]

File: pages.pt_BR/common/helm.md (hunk 1 above)
Comment (caduvieira) [reply pages.pt_BR/common/helm.md]: I changed. Please take a look when possible.

Comment (navarroaxel) [reply pages.pt_BR/common/helm.md]: @caduvieira Plase apply the same change to the pages `jar` and `kill`

Comment (caduvieira) [reply pages.pt_BR/common/helm.md]: Sorry my bad. Now it's done for all

**Review by navarroaxel — APPROVED** [review]

**Review by sbrl — APPROVED** [review]
Don't speak the language, but the syntax looks fine to me!

Thanks :slightly_smiling_face:

**Pull request merged**